manifests/0000_90_kube-scheduler-operator_04_servicemonitor-scheduler.yaml

```diff
@@ -93,6 +93,13 @@ spec:
     tlsConfig:
       caFile: /etc/prometheus/configmaps/serving-certs-ca-bundle/service-ca.crt
       serverName: scheduler.openshift-kube-scheduler.svc
+    metric_relabel_configs:
```

Comment:
you could use [`honorLabels: true`](https://github.com/prometheus-operator/prometheus-operator/blob/master/Documentation/api.md#endpoint) instead.

Reply:
nice